src/modules/config-panel/config-panel.html

```diff
@@ -25,19 +31,43 @@ <h1>Settings</h1>
       </div>
       <div class="card-body">
         <div class="form-check">
-          <label class="form-check-label"><input type="checkbox" class="form-check-input" checked.bind="allowUnauthorizedCertificates">Trust unauthorized certificates</label>
+          <label class="form-check-label"
+            ><input type="checkbox" class="form-check-input" checked.bind="allowUnauthorizedCertificates" />Trust
+            unauthorized certificates</label
+          >
```

Comment:
Gibts fürs BPMN Studio eigentlich auch ne prettier config?

```suggestion
          <label class="form-check-label">
            <input type="checkbox" class="form-check-input" checked.bind="allowUnauthorizedCertificates" />
            Trust unauthorized certificates
          </label>
```

Reply:
Ja :D die hat das so eingestellt 😂

Reply:
...
:bomb: